### PR TITLE
Unhide new migrated Catalog > Files Symfony page

### PR DESCRIPTION
**Pull request by matks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Unhide new migrated Catalog > Files Symfony page and plug legacy_links for redirections
| Type?         | new feature
| Category?     | BO
| BC breaks?    | yes, legacy controller files are removed
| Deprecations? | no
| Fixed ticket? | Fixes 1 item of https://github.com/PrestaShop/PrestaShop/issues/10550
| How to test?  | Please check that now all links to Catalog > Files redirect to migrated page instead of legacy page

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/16815)

### CODE REVIEW

**Comment by matks** [comment]
Thank you @RaimondasSapola for providing the legacy_links, I only needed to activate them !

**Review by jolelievre — APPROVED** [review]
Thank you @matks

**Review by PierreRambaud — APPROVED** [review]

**Comment by matks** [comment]
Thanks for review

**Comment by matks** [comment]
(I clicked on "close" instead of "merge", damned, gonna need to wait for travis again)

File: src/PrestaShopBundle/Resources/config/routing/admin/configure/advanced_parameters/webservice.yml [hunk 1]
```diff
@@ -38,7 +38,7 @@ admin_webservice_keys_edit:
         _legacy_controller: AdminWebservice
         _legacy_link: AdminWebservice:updatewebservice_account
         _legacy_parameters:
-            id_webservice_account: webserviceAccountId
+            id_webservice_account: webserviceKeyId
```
Comment (matks) [inline src/PrestaShopBundle/Resources/config/routing/admin/configure/advanced_parameters/webservice.yml:41]: 🤔 I dont remember adding this in my PR ?

Comment (matks) [reply src/PrestaShopBundle/Resources/config/routing/admin/configure/advanced_parameters/webservice.yml:41]: @jolelievre aaaaah 😅you added that

Ouf, I thought I was becoming crazy 😃

Comment (jolelievre) [reply src/PrestaShopBundle/Resources/config/routing/admin/configure/advanced_parameters/webservice.yml:41]: Yes I added a few tests in url coverter tests, which highlighted some wrong parameters ^^

**Comment by matks** [comment]
Thanks for review @PierreRambaud and for the fixes @jolelievre !

**Pull request merged**